### ISSUE TITLE
`python_version` in generated pytype.cfg is different from its python version

### ISSUE THREAD

**Issue by tamuhey** [google/pytype#270]
Thanks for this great tool!

I use pytype with virtualenv, and the python version is 3.7:
```
$ python --version
Python 3.7.2
```
and install pytype with pip:
```
$ python -m pip install pytype
```
and generate pytype.cfg:
```
$ pytype --generate-config pytype.cfg
```
but the `python_version` field of `pytype.cfg` is not 3.7, which is the version of venv Python:
```pytest.cfg
python_version = 3.6
```

Is it possible that make the  `python_version` field same as its venv Python version when `pytype.cfg` is generated?

**Comment by rchen152** [comment]
Thanks for using pytype! Looks like we currently have the target python version hard-coded here: https://github.com/google/pytype/blob/fddbad8c00a08b5d90acbbea8b25f7407ae619c7/pytype/tools/analyze_project/config.py#L55. I agree that it would be better to use the venv Python version instead; we should be able to get that with `sys.version_info[:2]`.

**Comment by tamuhey** [comment]
@rchen152 I wrote PR! #272

**Comment by rchen152** [comment]
@tamuhey Thanks for fixing this! The fix has been released in pytype version 2019.03.27.
